### DISCUSSION TITLE
Laser engraving memorial name plates

### DISCUSSION THREAD

**Discussion by BillD1382** [arkypita/LaserGRBL#1395] [Q&A]
We have a new laser master 2 from Ortur. The primary use for this machine will be engraving 1 inch by 3 inch bras name plates for our Memorial board for members of our Legion Post who have passed away.
I have not been able to find any directions on how to do this single engraving of name plates and am wondering if you could supply some directions on how I would accomplish this. We will be using either all brass or brass with black so names stand out.
I have only been able to find directions to print multiple name plates not just one at a time.

Finally if possible could you recommend a place to purchase blank name plates in bulk. with our new engraver we will have approximately 75 name plates to engrave.

Thank you in advance for any information you can supply

**Comment by StuartB4** [comment]
If you want to engrave them you will need a fiber laser. You can engrave in to a coating on top of the brass. But you can't engrave metal with a diode laser.
